### ISSUE TITLE
Should it work with a copy of Tribsoft's Linux port?

### ISSUE THREAD

**Issue by mmazza72** [ja2-stracciatella/ja2-stracciatella#365]
running v0.15.0 on Debian stretch amd64.
game quits with this message:
```
ERROR: caught unhandled exception:
Opening file 'loadscreens/titletext.sti' failed
Creating an emergency savegame failed.
Please report this error with a description of the circumstances.
```
the file above is nowhere to be found in the game installation.

**Comment by selaux** [comment]
I personally never tried if it works, but if it doesn't, I would consider that a bug. `tiletext.sti` is actually then menu buttons displayed in the main menu. Does it differ from the windows version in any way?

**Comment by selaux** [comment]
You could also try to extract your game sources using https://github.com/ja2-stracciatella/ja2-open-toolset. That way we can see what is actually in your game resources.

**Comment by mmazza72** [comment]
I extracted the sources and located the missing file.
My copy of the game (which I can't compare to the Windows version) is split in two cds.
When I installed from the first cd (full install) it never required the second cd, so it's supposed to stay in the drive.
The second cd contains these files under data/:
```
loadscreens.slf  maps.slf  movie.dat  npc_speech.slf  speech.slf
```
loadscreens.slf contains the missing file.
If I copy it under the local installation the menu loads fine.

Now what confuses me is that the other four files exist identically in the local installation, so the only need for the second cd is this 26M file.
Why not copy it over during a full install?

Anyway, the issue is fixed by copying loadscreens.slf from the cd labelled "game disk" to the installation.
